app/controllers/comments_controller.rb

```diff
@@ -76,10 +76,10 @@ def send_notification(message, user_id)
     Notification.create(
       body: format(message, poster: current_user.username,
                             target: "#{@target.display_title} (ID: #{@target.id})"),
-      source_type: 'Submission',
-      source_id: @target.id,
+      source_type: 'Comment',
+      source_id: @comment.id,
       user_id: user_id,
-      url: submission_path(@target)
+      url: @comment.url
```

Comment:
Comment makes sense as a source type but @comment.url is an invalid reference, is it not?

Reply:
wait hold on i just realized something
this is fine for now but it won't be sustainable if @comment.url changes to support other things like submissions and whatnot
we'd need a migration for it to reliably point to the well, pointed content
i remember this was what i had issue with and why i said "leave this for DAD 2.0"

Reply:
ah wait i just realized
source_type